webpack.config.js

```diff
@@ -1,21 +1,17 @@
 const path = require('path');
```

Comment:
a merge from master should also prevent changes to this file
basically, the only file changed in this PR should be garmin.ts